compiler/rustc_passes/messages.ftl

```diff
@@ -46,6 +46,11 @@ passes_attr_crate_level =
     .suggestion = to apply to the crate, use an inner attribute
     .note = read <https://doc.rust-lang.org/nightly/rustdoc/the-doc-attribute.html#at-the-crate-level> for more information
 
+passes_attr_mod_level =
+    this attribute can only be applied at module level
+    .suggestion = to apply to the crate, use an inner attribute
```

Comment:
```suggestion
    .suggestion = to apply to the crate, use an inner attribute at the crate level
```

Reply:
We can use inner attributes everywhere, without precising "at crate level" it doesn't really make sense.

Reply:
Indeed, applied your suggestion.